schema.sql

```diff
@@ -11,3 +11,98 @@ CREATE TABLE animals (
 
 ALTER TABLE animals
 ADD COLUMN species VARCHAR(255);
+
+
+-- Create a table named owners
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name TEXT,
```

Comment:
- [ OPTIONAL] Well done creating the tables, I will just recommend to use `varchar` instead of `text` , you can have further informations about the topic [here](https://www.postgresql.org/docs/current/datatype-character.html).